fiduswriter/base/root_urls.py

```diff
@@ -48,6 +48,14 @@
         'path': 'sw.js'
     }),
 
+    url('^sw2.js$', static_serve, {
```

Comment:
@anandksalien Are we using both sw2.js and sw.js or is this replacing sw.js?

Reply:
We need the transpiled version of sw.js and we import it in sw2.js. so we would be needing both sw.js and sw2.js.(sw.js needs to be accessible via an URL endpoint)